legal-api/report-templates/correction.html

```diff
@@ -51,5 +51,10 @@
          [[correction/rulesMemorandum.html]]
          [[correction/resolution.html]]
       </div>
+      <div class="container business-details-table" >
+         <div class="bold"><span class="details-header-text"> Correction Detail</span><span class="correction-label-added">ADDED</span></div>
+         <div class="pt-2">Correction filed by {{correction.type}} on {{effective_date_time}}</div>
```

Comment:
What are the different values for `type`?

In the UI, if the current user is staff then we show the staff name here; if the current user is not staff then we show "by Registry Staff".

![image](https://github.com/user-attachments/assets/793a5c55-4547-4297-8f45-023dfd4ab04b)

In this case, to prevent outputs from accidentally showing a staff's name, I think this can simply be hard-coded to "by Registry Staff" (or "by Registries Staff" if you follow the UI example above).

Reply:
We talked about this. Resolving.